apim/3.x/templates/api/api-configmap.yaml

```diff
@@ -51,6 +51,7 @@ data:
         {{- if (eq .Values.mongo.auth.enabled true) }}
         username: {{ .Values.mongo.auth.username }}
         password: {{ .Values.mongo.auth.password }}
+        authSource: {{ .Values.mongo.auth.source | default "admin" }}
```

Comment:
Just wondering why we would want to move "admin" as default authSource ?
To be consistent with the code, I would prefer to keep `gravitee`.
See https://github.com/gravitee-io/gravitee-repository-mongodb/blob/master/src/main/java/io/gravitee/repository/mongodb/common/MongoFactory.java#L205

Reply:
I was confused for a second. the `values.yaml` has `admin` listed but the code uses `gravitee`. I deleted that commit now.

Reply:
@brasseld , should I change the value of the auth source in the `values.yaml` to `gravitee` as well for consistency?

Reply:
ok, now I understand. So you're right, `admin` should be set by default, otherwise it will be no more compatible with previous chart versions.

Reply:
I think `gravitee` is the right default though, because if you did not specify `.Values.mongo.auth.source` in your `values.yaml`, `gravitee` is the one that is used behind the scenes. In all other cases, what you specify is what is used.

Reply:
Indeed, good point!

So I think that's good.